cmd/promtool/main.go

```diff
@@ -353,8 +353,11 @@ func checkConfig(agentMode bool, filename string) ([]string, error) {
 	}
 
 	for _, scfg := range cfg.ScrapeConfigs {
-		if err := checkFileExists(scfg.HTTPClientConfig.BearerTokenFile); err != nil {
-			return nil, errors.Wrapf(err, "error checking bearer token file %q", scfg.HTTPClientConfig.BearerTokenFile)
+		authzSet := scfg.HTTPClientConfig.Authorization != nil
+		if authzSet {
+			if err := checkFileExists(scfg.HTTPClientConfig.Authorization.CredentialsFile); err != nil {
+				return nil, errors.Wrapf(err, "error checking authorization credentials file %q", scfg.HTTPClientConfig.Authorization.CredentialsFile)
+			}
 		}
```

Comment:
curious if we should have an `else` block that still supports bearer token? Not sure if anyone would still be using that?

Reply:
common is moving `BearerTokenFile` into `CredentialsFile`, so it's not possible that `BearerTokenFile` is being used.

https://github.com/prometheus/common/blob/ce7006ede1b27960d267508d7ae0af2f51422610/config/http_config.go#L243-L252